raiden/tests/scenarios/funding-channel-scenarios.yaml

```diff
@@ -0,0 +1,226 @@
+            - assert: {from: 0, to: 1, total_deposit: 99_000_000_000_000_000_000, balance: 99_000_000_000_000_000_000, state: "opened"}
+
+      - serial:
+          name: Scenario F12 (Depositing incredibly huge number)
+          tasks:
+           - assert: {from: 0, to: 1, total_deposit: 99_000_000_000_000_000_000, balance: 99_000_000_000_000_000_000, state: "opened"}
+           - deposit: {from: 0, to: 1, total_deposit: 99_000_000_000_000_000_000_000_000_000_000_000_000_000_000_000_000_000_000_000_000_000_000_000_000, expected_http_status: 402}
+           - assert: {from: 0, to: 1, total_deposit: 99_000_000_000_000_000_000, balance: 99_000_000_000_000_000_000, state: "opened"}
+
+      - serial:
+          name: Scenario F13 (Depositing smallest possible number)
+          tasks:
+            - assert: {from: 0, to: 1, total_deposit: 99_000_000_000_000_000_000, balance: 99_000_000_000_000_000_000, state: "opened"}
+            - deposit: {from: 0, to: 1, total_deposit: 99_000_000_000_000_000_001, expected_http_status: 200}
+            - assert: {from: 0, to: 1, total_deposit: 99_000_000_000_000_000_001, balance: 99_000_000_000_000_000_001, state: "opened"}
+
+      - serial:
+          name: Scenario F16 (Depositing more than available on-chain balance of node 1)
+          tasks:
+            - assert: {from: 0, to: 1, total_deposit: 99_000_000_000_000_000_001, balance: 99_000_000_000_000_000_001, state: "opened"}
+            - deposit: {from: 0, to: 1, total_deposit: 100_99_999_000_000_000_000_001, expected_http_status: 402}
+            - assert: {from: 0, to: 1, total_deposit: 99_000_000_000_000_000_001, balance: 99_000_000_000_000_000_001, state: "opened"}
+
+      - parallel:
+          name: Scenario F20 (Deposit and then kill node before the transaction was submitted to the blockchain)
+          tasks:
+            - serial:
+                tasks:
+                  - assert: {from: 0, to: 1, total_deposit: 99_000_000_000_000_000_001, balance: 99_000_000_000_000_000_001, state: "opened"}
+                  - deposit: {from: 0, to: 1, total_deposit: 99_100_000_000_000_000_000, expected_http_status: 500}
+                tasks:
+                  - wait: 1
+                  - stop_node: 1
+                  - wait: 600
+                  - start_node: 1
+                  - assert: {from: 0, to: 1, total_deposit: 99_000_000_000_000_000_001, balance: 99_000_000_000_000_000_001, state: "opened"}
+
+      - parallel:
+          name: Scenario F21 (Deposit and then kill node after the transaction is submitted to the blockchain)
+          tasks:
+            - serial:
+                tasks:
+                  - assert: {from: 0, to: 1, total_deposit: 99_000_000_000_000_000_001, balance: 99_000_000_000_000_000_001, state: "opened"}
+                  - deposit: {from: 0, to: 1, total_deposit: 99_100_000_000_000_000_000, expected_http_status: 200}
+            - serial:
+                tasks:
+                  - wait: 500
```

Comment:
Do you need to wait that much?

Reply:
We want to make sure that the "deposit" task is submitted for sure. Since we can not listen for events at the moment, I'd rather wait for 1 more minute than not testing the right thing. Do you agree?